lua/astrocommunity/bars-and-lines/dropbar-nvim/init.lua

```diff
@@ -7,6 +7,7 @@ return {
   },
   {
     "catppuccin/nvim",
+    name = "catppuccin",
```

Comment:
```suggestion
     "catppuccin",
```

I think maybe in this case we should just replace `catppuccin/nvim` with the shorthanded `catppuccin` rather than giving the repo and name each time for it to be a bit cleaner. We do try to avoid the shorthand notation but in the case of a custom name it could make more sense

Reply:
hey, thanks for looking into this. i have done the change you suggested!